.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +0,0 @@
-<!--
-  Thanks for submitting a pull request!
-  Please provide enough information so that others can review your pull request.
--->
-
-**Summary**
```

Comment:
do you think this isn't useful?

Reply:
Not so much in the way it was anyway. We can work on a better one once we know what to expect in that template.